.github/workflows/pluto_build.yml

```diff
@@ -48,13 +80,13 @@ jobs:
         shell: bash
         working-directory: ./db-pluto/pluto_build
         run: ./01_dataloading.sh
-        if: inputs.minor_version == false
+        if: needs.version.outputs.is_minor_version == 'false'
```

Comment:
I think longer term we might just want to specify version name at top level, then have this sort of logic within the pluto scripts themselves. But for now think this is fine